File: kildey004/programacaobasica/questao5.py
print (" KR seguros")

print (" >> Avaliacao dos candidatos << ")
contratoS = float(2300)
contratoC = float(3500)

qs = float(input("Qtd de contratos simples: "))
qc = float(input("Qtd de contratos completos: "))

csT = contratoS * qs
ccT = contratoC * qc

total = csT + ccT
lucro = total / 10 

print ("lucro obtido: R$" + str(lucro))

File: kildey004/programacaobasica/Questao5.java
import java.util.Scanner;
public class Questao5 {
    public static void main (String[] args) {
        double quantidadeS;
        double quantidadeC;
        double csT;
        double ccT;
        double total;
        double lucro;
        Scanner s;

        System.out.println("KR seguros, Avaliacao dos candidatos");
        
        s = new Scanner(System.in);
        System.out.println("Qtd de contratos simples: ");
        quantidadeS = s.nextDouble();
        
        
        System.out.println("Qtd de contratoscompletos: ");
        quantidadeC = s.nextDouble();

        csT = quantidadeS * 2300;
        ccT = quantidadeC * 3500;

        total = csT + ccT;
        lucro = total / 10;

        System.out.println("Lucro obtido: R$" + lucro);
    }
}
